src/Faker/Provider/hr_HR/Address.php

```diff
@@ -0,0 +1,73 @@
+<?php
+
+namespace Faker\Provider\hr_HR;
+
+class Address extends \Faker\Provider\Address
+{
+    protected static $streetPrefix = array(
+        'Trg', 'Ulica', 'Avenija'
+    );
+
+    protected static $street = array(
+        'Ante Starčevića', 'bana Jelačića', 'Ive Andrića', 'Gojka Šuška', 'Vladimira Nazora', 'Augusta Cesareca',
```

Comment:
Too small set of data. Faker needs to provide random street names, not the same names avery 10 calls.